sig-windows/annual-report-2021.md

```diff
@@ -3,120 +3,82 @@
+     - (1.23) https://github.com/kubernetes/enhancements/blob/master/keps/sig-windows/2802-identify-windows-pods-apiserver-admission/kep.yaml
    - Alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.alpha
-   - Pre-alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md)
+     - (1.22) [1981 - Windows Privileged Container Support](https://github.com/kubernetes/enhancements/blob/master/keps/sig-windows/1981-windows-privileged-container-support/README.md)
+     - (1.23) [2802 -Identify Windows pods at API admission level authoritatively](https://github.com/kubernetes/enhancements/tree/master/keps/sig-windows/2802-identify-windows-pods-apiserver-admission/README.md)
+   - Pre-alpha (Targeting 1.24)
+     - [2578 - Windows Conformance](https://github.com/kubernetes/enhancements/blob/master/keps/sig-windows/2578-windows-conformance/kep.yaml)
 
 ## Project health
 
 1. What areas and/or subprojects does your group need the most help with?
    Any areas with 2 or fewer OWNERs? (link to more details)
-
-   -
-   -
-   -
-
+   - csi-proxy and storage: this seems like an underserved area for windows https://github.com/kubernetes-csi/csi-proxy (meeting info is there).
 2. What metrics/community health stats does your group care about and/or measure?
-
-   -
-   -
-   -
-
+   - stars for ksigs/windows specific repos
+     - sig-windows-dev-tools   
+       - https://github.com/kubernetes-sigs/sig-windows-dev-tools -> up to 46, represents interest
+     - sig-windows-tools
+       - represents people trying to install windows on k8s nodes
+     - windows-gmsa
+       - represents enterprises integrating windows pods into GMSA
 3. Does your [CONTRIBUTING.md] help **new** contributors engage with your group specifically by pointing
    to activities or programs that provide useful context or allow easy participation?
-
-   -
-
+   - yes
 4. If your group has special training, requirements for reviewers/approvers, or processes beyond the general [contributor guide],
    does your [CONTRIBUTING.md] document those to help **existing** contributors grow throughout the [contributor ladder]?
-
-   -
+   - yes
 
 5. Does the group have contributors from multiple companies/affiliations?
-
-   -
+   - yes
 
 6. Are there ways end users/companies can contribute that they currently are not?
    If one of those ways is more full time support, what would they work on and why?
-
-   -
-   -
+   - testing hostProcess implementations on several windows apps
+   - improving our dev tools environment to grow the community
+   - hardening the CSI proxy and CSI support ecosystem
+   - performance testing Kubernetes on Windows extensively and publishing results
```

Comment:
where should someone who sees these and wants to help go? are there topics or tracking issues for these?

Reply:
added specific target of blog post